app/controllers/admin/items_controller.rb

```diff
@@ -22,10 +23,12 @@ def create
   def edit
```

Comment:
As it stands we cant protect the create actions without turning the object variable into an instance variable.